lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -115,6 +114,13 @@ class Core
     ["-g", "--global"] => [ false, "Operate on global datastore variables"]
   )
 
+  VALID_PARAMS =
```

Comment:
```suggestion
  VALID_SESSION_SEARCH_PARAMS =
```

Reply:
And can we use `private_constant :VALID_SESSION_SEARCH_PARAMS` too to ensure this doesn't become a value part of this classes public API :+1:

https://aaronlasseigne.com/2016/10/26/know-ruby-private_constant/